recipes/professor/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "Professor" %}
+{% set version = "2.3.4" %}
+{% set tag = name|lower ~ "-" ~ version %}
+{% set repo = "https://gitlab.com/hepcedar/" ~ name|lower %}
+{% set home = "https://" ~ name|lower ~ ".hepforge.org" %}
```

Comment:
It's best to avoid being too clever with jinja2 as it makes the recipe hard to analyse:
```suggestion
```